### ISSUE TITLE
RTCPeerConnection incorrect for Safari

### ISSUE THREAD

**Issue by brentfulgham** [Modernizr/Modernizr#2559]
Modernizr 3.5.0 believes that WebKit (Safari) does not support RTCPeerConnection because we expose an unprefixed version. Modernizr expects a webkit-prefixed version:

From WebInspector on Safari 13:
```
> window["webkitRTCPeerConnection"]
< undefined

> window["RTCPeerConnection"]
< function RTCPeerConnection() {
    [native code]
}

```
This leads to many sites losing RTC features for Safari/WebKit clients.

**Comment by rejas** [comment]
When opening https://modernizr.github.io/Modernizr/test/integration.html (v3.9.1) on my safari 13.1 I see correctly "peerconnection: true" Can you check that page too?

**Comment by brentfulgham** [comment]
The test page works correctly. Digging further, it looks like the site I was investigating (https://www.webliero.com) must have configured modernizr to use DOM prefixes. We no longer prefix RTCPeerConnection, so it fails to recognize the support.

The site may be looking for the prefixed version because they are using the deprecated, Chrome-specific callback-based APIs, which we don't support.

I'll close as behaves correctly.

**Comment by rejas** [comment]
Ehm, that site you mentioned says to me it is not working because safari is not supporting `datachannel` according to my safari...

**Comment by rejas** [comment]
which got fixed in v3.8.0 see https://github.com/Modernizr/Modernizr/issues/2307 

so if they upgrade to the latest version all should be well....
